server/config/config.go

```diff
@@ -43,10 +43,11 @@ const (
 	defaultMinScanLimit    int  = 20
 	defaultIDAllocatorStep uint = 20
 
-	defaultClusterName              = "defaultCluster"
-	defaultClusterNodeCount         = 2
-	defaultClusterReplicationFactor = 1
-	defaultClusterShardTotal        = 8
+	defaultClusterName                     = "defaultCluster"
+	defaultClusterNodeCount                = 2
+	defaultClusterReplicationFactor        = 1
+	defaultClusterShardTotal               = 8
+	defaultPartitionTableProportionOfNodes = float32(0.5)
```

Comment:
I guess no need to do explicit conversion:
```suggestion
	defaultPartitionTableProportionOfNodes = 0.5
```

Remember to fix the same problems at other places.